bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,36 @@
+def swap(arr, i, j):
+    temp = arr[i]
+    arr[i] = arr[j]
+    arr[j] = temp
+
+
+def bubble_sort(arr, n):
+    i = 0
+    j = 0
+    swapped = False
+    for i in range(n):
+        for j in range(n-i-1):
+            if (arr[j] > arr[j+1]):
+                swap(arr, j, j+1)
+                swapped = True
+			
+        if (swapped == False):
+            break
+	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp1z8jgz6r/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmp1z8jgz6r/bubble_sort/Python/bubble_sort.py
@@ -16,7 +16,7 @@
 			
         if (swapped == False):
             break
-	
+
 
 
 #Function to print an array
```